src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -153,30 +176,29 @@ def generate_MCP_1_default_config(self, filename):
                     'certFile': 'server_b.cert',
                     'keyFile': 'server_b.key',
                     'certificateHash': 'will be provided in future',
-                    'cbsdRecord': [sas_th_1_cbsd_iter_0, sas_th_1_cbsd_iter_1, sas_th_1_cbsd_iter_2],
+                    'sasThIterationData': [sas_th_1_iteration_0_data,sas_th_1_iteration_1_data],
                     'fileServePath': 'sas_harness_1/'
                 }
         # Create the actual config.
         iter1_config = {
-            'cbsdRecords': cbsd_records_iter_1,
+            'cbsdRequests': [cbsd_records_iter_1_domain_proxy_0,cbsd_records_iter_1_domain_proxy_1],
             'protectedEntities': protected_entities_iter_1,
-            'dpaActiveList': [dpa_1, dpa_2],
-            'dpaDeactiveList': [],
-            'deltaIap': 2
+            'dpaActivationList': [dpa_1, dpa_2],
+            'dpaDeactivationList': [],
            }
         iter2_config = {
-            'cbsdRecords': cbsd_records_iter_2,
+            'cbsdRequests': [cbsd_records_iter_2_domain_proxy_0,cbsd_records_iter_2_domain_proxy_1],
             'protectedEntities': protected_entities_iter_2,
-            'dpaActiveList': [dpa_3],
-            'dpaDeactiveList': [dpa_1],
-            'deltaIap': 2
+            'dpaActivationList': [dpa_3],
+            'dpaDeactivationList': [dpa_1],
            }
 
         config = {
             'iterationData': [iter1_config, iter2_config],
-            'sasHarnesses': [sas_harness_0, sas_harness_1],
+            'sasHarnessesConfigs': [sas_harness_0, sas_harness_1],
             'domainProxyConfig': [{'cert': 'dp_1_client.cert', 'key': 'dp_1_client.key'},
```

Comment:
The variable name should be plural to indicate that it's a list of DP configs.

Reply:
Modified   to domainProxyConfigs